fdbserver/Knobs.cpp

```diff
@@ -317,6 +317,7 @@ ServerKnobs::ServerKnobs(bool randomize, ClientKnobs* clientKnobs) {
 	init( ENFORCED_MIN_RECOVERY_DURATION,                       0.085 ); if( shortRecoveryDuration ) ENFORCED_MIN_RECOVERY_DURATION = 0.01;
 	init( REQUIRED_MIN_RECOVERY_DURATION,                       0.080 ); if( shortRecoveryDuration ) REQUIRED_MIN_RECOVERY_DURATION = 0.01;
 	init( ALWAYS_CAUSAL_READ_RISKY,                             false );
+	init( MAX_COMMIT_UPDATES,                                  100000 ); if( randomize && BUGGIFY ) MAX_COMMIT_UPDATES = 1;
```

Comment:
I think the idea seems reasonable, though I wonder if this number needs to be so high. If I'm reading the code correctly, it will issue this request at most once per 50ms, so having 100,000 outstanding would correspond to starting these requests for ~80 minutes without response. Or alternatively, if we are just issuing these requests faster than we are completing them, it still seems like a pretty big backlog to carry. Is there value in having so many of these in flight?

Reply:
this function is also called when getting read versions when causal_read_risky is not enabled, so I choose an overly large number on purpose to prevent effecting production